platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/PlatformCoordinator.java

```diff
@@ -74,14 +82,16 @@ public void clear() {
         inOrderLinkerWiring.flushRunnable().run();
         linkedEventIntakeWiring.flushRunnable().run();
 
-        // once everything has been flushed out of the system, it's safe to unpause the linked event intake
+        // once everything has been flushed out of the system, it's safe to unpause event intake and creation
         linkedEventIntakeWiring.pauseInput().inject(false);
+        eventCreationManagerWiring.pauseInput().inject(false);
 
         // data is no longer moving through the system. clear all the internal data structures in the wiring objects
         eventDeduplicatorWiring.clearInput().inject(new ClearTrigger());
         eventDeduplicatorWiring.flushRunnable().run();
         orphanBufferWiring.clearInput().inject(new ClearTrigger());
         orphanBufferWiring.flushRunnable().run();
+        eventCreationManagerWiring.flush();
```

Comment:
What's the intention of this flush?

Reply:
I've updated this code based on our discussion.